assets/templates/general/widget_search_select.html

```diff
@@ -0,0 +1,37 @@
+<input id="{{ widget.name }}-display"
+       name="{{ widget.name }}-display"
+        {% if widget.value != None %}
+            value="{{ widget.value_name }}"
+        {% endif %}
+        {% include "django/forms/widgets/attrs.html" %}
+>
+
+<input id="{{ widget.name }}"
+       type="hidden"
+       name="{{ widget.name }}"
+        {% if widget.value != None %}
+            value="{{ widget.value|stringformat:'s' }}"
+        {% endif %}
+
+>
+
+<script>
+    document.addEventListener("DOMContentLoaded", function (event) {
+        let searchfield = document.getElementById("{{ widget.name }}-display");
+        let numberfield = document.getElementById("{{ widget.name }}");
+
+        // Enable autocomplete
+        new Awesomplete(searchfield, {
+            list: [{% for entry in widget.queryset %}["{{ entry }}","{{ entry.id }}"],{% endfor %}],
+            replace: function (text) {
+                this.input.value = text.label;
+            },
+            autoFirst: true,
+        });
+
+        // Update internal button when user selects someone
```

Comment:
Outdated comment

Reply:
Yup, changed it in last commit:
// Update the selected instance-value